src/api/MockNetworkInterface.js

```diff
@@ -0,0 +1,14 @@
+const { mockServer } = require("graphql-tools");
+const { print } = require("graphql/language/printer");
+
+class MockNetworkInterface {
+  constructor(schema, mocks = {}) {
+    this.mockServer = mockServer(schema, mocks);
+  }
+
+  query(request) {
+    return this.mockServer.query(print(request.query), request.variables);
+  }
+}
+
+module.exports = MockNetworkInterface;
```

Comment:
Shouldn't this be dropped in favour of the same module from `eq-author-mock-api`?